src/mock_crust/support.rs

```diff
@@ -0,0 +1,449 @@
+// Copyright 2016 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under (1) the MaidSafe.net Commercial License,
+// version 1.0 or later, or (2) The General Public License (GPL), version 3, depending on which
+// licence you accepted on initial access to the Software (the "Licences").
+//
+// By contributing code to the SAFE Network Software, or to this project generally, you agree to be
+// bound by the terms of the MaidSafe Contributor Agreement, version 1.0.  This, along with the
+// Licenses can be found in the root directory of this project at LICENSE, COPYING and CONTRIBUTOR.
+//
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.
+//
+// Please review the Licences for the specific language governing permissions and limitations
+// relating to use of the SAFE Network Software.
+
+// It seems that code used only in tests is considered unused by rust.
+#![allow(unused)]
```

Comment:
No, it doesn't work. Without `#![allow(unused)]` it complains about unused stuff. With `#![cfg(test)]` it complains about unresolved imports. It seems that the `test` and `feature = "..."` cfgs interact in some weird and confusing ways. Don't want to spend too much time on this, so I'd keep it as is. We should probably extract mock crust into a separate crate or move it to the crust create, as that would solve this issue (I think).